gammapy/irf/background.py

```diff
@@ -254,7 +255,7 @@ def to_3d(self):
 
         axes = MapAxes([self.axes["energy"], fov_lon, fov_lat])
         coords = axes.get_coord()
-        offset = np.sqrt(coords["fov_lat"] ** 2 + coords["fov_lon"] ** 2)
+        offset = angular_separation(0, 0, coords["fov_lon"], coords["fov_lat"])
```

Comment:
A very naive question: should we put `0` or `0*u.deg` ? I have not made the test...

Reply:
For the center of the axes, it should be the same, as floats are transformed in `Quantitiy` instances by `astropy` assuming they are in radians.

Reply:
I'd pass `0 * u.rad` for clarity

Reply:
Thanks for the feedback, I added the units for clarity